Local build should not require npm

Repro:

Vanilla OS, clone

```
./build.sh
```

Result:

```
/home/aspnet/src/aspire/playground/BrowserTelemetry/BrowserTelemetry.Web/BrowserTelemetry.Web.csproj(25,5): error MSB3073: The command "npm install" exited with code 127.
```

--- a/playground/BrowserTelemetry/BrowserTelemetry.Web/BrowserTelemetry.Web.csproj
+++ b/playground/BrowserTelemetry/BrowserTelemetry.Web/BrowserTelemetry.Web.csproj
@@ -32,7 +32,7 @@
       2. Run npm run build before building the .NET project.
       MSBuild runs NpmInstall before this task because of the DependsOnTargets attribute.
    -->
-  <Target Name="NpmRunBuild" DependsOnTargets="NpmInstall" BeforeTargets="BeforeBuild" Condition="'$(Configuration)' == 'Debug' and '$(NpmUnavailable)' != 'true' ">
+  <Target Name="NpmRunBuild" DependsOnTargets="NpmInstall" Condition="'$(Configuration)' == 'Debug' and '$(NpmUnavailable)' != 'true' ">
     <Exec Command="npm run build" />
   </Target>
 

--- a/playground/BrowserTelemetry/BrowserTelemetry.Web/README.md
+++ b/playground/BrowserTelemetry/BrowserTelemetry.Web/README.md
@@ -1,0 +1,10 @@
+# Updating web assets
+
+When changes to the `./package.json` or `./Scripts/index.js` are made the `./wwwwroot/scripts/bundle.js` file needs to be updated by running the following command:
+
+```console
+dotnet build /t:NpmRunBuild
+```
+
+This will install the required npm packages and run the webpack command generating this file.
+Note that the `package-lock.json`

